t/resp_body_filters.t

```diff
@@ -10,13 +10,14 @@ sub lc {
 }
 
 my @block_list = blocks();
-my $i = 0;  # Use $i to make copy/paste of tests easier.
 
-my $html = "<html><head><title>Google</title></head><body>Search me...</body></html>";
-my $raw_res="HTTP/1.0 200 OK\r\nDate: Fri, 31 Dec 1999 23:59:59 GMT\r\nContent-Type: text/html\r\nContent-Length: ".length($html)."\r\n\r\n".$html;
-my ( $res, $raw_headers, $left ) = Test::Nginx::Socket::parse_response("name", $raw_res, 0);
-Test::Nginx::Socket::transform_response_body($block_list[$i], $res);
-Test::Nginx::Socket::check_response_body($block_list[$i], $res, undef, 0, 0, 0);
+for (my $i = 0; $i < 7; $i++) {
```

Comment:
We shouldn't hard-code the test block count, 7, here. Use the following form is better:

```perl
for my $block (blocks) {
}
```